config/api.yml

```diff
@@ -200,6 +200,29 @@
     :options:
     - :primary
     :verbs: *gd
+  :authentications:
+    :description: Authentications
+    :options:
+    - :collection
+    :verbs: *gpd
+    :klass: Authentication
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: embedded_automation_manager_credentials_view
```

Comment:
@abellotti how should we handle this identifier? We are using the `Authentication` model, but only have this identifier currently. Is it fine as is, and as needed we will add additional ones via #13827 ?